packages/account/src/utils/Types.ts

```diff
@@ -0,0 +1,15 @@
+import { Web3Provider } from '@ethersproject/providers'
+import { Signer } from 'ethers'
+export type SmartAccountConfig = {
+    epAddress: string,
+    bundlerUrl?: string
+}
+export type BiconomySmartAccountConfig = {
```

Comment:
is this the config used to initialise the sdk entrypoint class or SmartAccount standalone class from account. package?

Reply:
its specific to BiconomySmartAccount class whoever create their own class as add their own object that will be accepted in constructor